api/db/changelog/db.changelog-233-add-user-initial-credit-expiration-notification-column.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="evrii" id="changelog-232-add-user_initial_credits_expiration">
+    <addColumn tableName="user_initial_credits_expiration">
+      <column name="notification_status" type="tinyint" defaultValueNumeric="0">
+        <constraints nullable="false"/>
```

Comment:
Using int to represent enum is the old way and we agree to use `enum` instead 

Example: https://github.com/all-of-us/workbench/blob/2ded44a0cb32fb6eed7d66808e03ccc73c75ba87/api/db/changelog/db.changelog-167-add-access-module-tables.xml#L12